README.md

```diff
@@ -780,9 +784,18 @@ model = "stabilityai/stablelm-tuned-alpha-7b"
 
 | Argument | Type             | Default | Description                                                                                      |
 | -------- | ---------------- | ------- | ------------------------------------------------------------------------------------------------ |
-| `model`  | `str`            |         | The name of a StableLM model that is supported.                                                  |
+| `model`  | `str`            |         | The name of a Dolly model that is supported.                                                     |
```

Comment:
Merge error? IIRC we had already fixed this before - this edit should be reverted.

Reply:
Likely - fixed in https://github.com/explosion/spacy-llm/pull/151/commits/2732d5993f7afd002270d036731d07d9c0504a22.